### ISSUE TITLE
NFT shouldn't be inited during auction

### ISSUE THREAD

**Issue by tvorogme** [TelegramMessenger/telemint#66]
According to [standard](https://github.com/ton-blockchain/TEPs/blob/master/text/0062-nft-standard.md) in get_nft_data - `init? - if not zero, then this NFT is fully initialized and ready for interaction.`, but if it's in auction state it's not ready for interaction, I guess? Because transfer is blocked while auction cell is not null https://github.com/TelegramMessenger/telemint/blob/26376dec9577cd210c8c8d11509f7f87dae720c8/func/nft-item.fc#L212

https://github.com/TelegramMessenger/telemint/blob/main/func/nft-item.fc#L281

**Comment by arseny30** [comment]
It is unclear from the standard what the desired behavior is. The nft itself is initialized. It has content. It even will respond to get_static_data.
For example, [this smart contract](https://github.com/ton-blockchain/dns-contract/blob/main/func/nft-item.fc#L44) almost always returns -1.

**Comment by tvorogme** [comment]
Sure, but as for backends (as I think) it'll much easier to run one get method, to know can we use NFTs in our APPs or not. 

All dApps uses transfer method to interact with it: any games, marketplaces, etc. 

It's sad, that official DNS contract have such behavior, but I personally think that we don't need to populate such behavior further

**Comment by tvorogme** [comment]
Otherwise, you will have to download every time NFT data&cod and run TVM to check the transfers locally to see if you can use them in your apes or not. This is very uncomfortable, as for me, you'll need to adapt to each NFT instead trust one bit in GET method 🤷‍♂️

**Comment by tvorogme** [comment]
Ah, owner is zero address on this stage, sorry, all OK, this is not needed

**Comment by EmelyanenkoK** [comment]
I don't think that it is a issue to show NFT as initiated when it is indeed initiated, but not yet found it's owner. It should not create a problem, since nobody can transfer this NFT anyway.

But, what about secondary auctions? NFT is not transferrable but `get_nft_data` won't give any hint about that (thus wallets won't correctly handle this situation by default). Maybe it is reasonable to show owner of NFT as null when auction is active?
